app/components/team/Index.js

```diff
@@ -61,19 +124,117 @@ class TeamIndex extends React.PureComponent {
     }
   }
 
+  renderTeamSearchInfo() {
+    const { organization: { teams }, relay: { variables: { teamSearch } } } = this.props;
+
+    if (teamSearch && teams) {
+      return (
+        <div className="bg-silver semi-bold py2 px3">
+          <small className="dark-gray">
+            {formatNumber(teams.count)} matching teams
```

Comment:
ooh, it occurs to me we haven’t done pluralisation here (or in the Member list! (or the Agent list!))